app/index.html

```diff
@@ -38,6 +38,33 @@
               <!--<pre>{{ userForm['user.email'] | json }}</pre>-->
             </div>
           </div>
+
+          <div class="form-group" af-field-wrap af-model-path="user.gender" has-error>
+            <label for="male" class="control-label col-md-2">{{user.gender}} Gender*</label>
+            <div class="col-md-4">
+              <div class="radio">
+                <label>
+                  <input type="radio" id="male" name="user.gender" ng-model="user.gender" value="male" af-field required /> Male
+                </label>
+              </div>
+              <div class="radio">
+                <label>
+                  <input type="radio" id="female" name="user.gender" ng-model="user.gender" value="female" af-field required /> Female
+                </label>
+              </div>
+              <div class="radio">
+                <label>
+                  <input type="radio" id="other" name="user.gender" ng-model="user.gender" value="other" af-field required /> Other, namely:
+                  <div af-field-wrap af-model-path="user.otherGender" has-error>
+                    <input type="text" class="form-control" name="user.otherGender" ng-model="user.otherGender" af-field af-trigger="user.gender === 'other'" ng-required="user.gender === 'other'" />
+                    <div af-error></div>
+                    <!--<pre>{{userForm['user.otherGender'] | json}}</pre>-->
```

Comment:
Ja eerder in de file ook, is ter debugging. In een andere branch heb ik dat een beetje opgeruimd, dus doe ik liever in 1x (als deze er bij gemerged wordt)